core/services/chainlink/application.go

```diff
@@ -206,6 +207,13 @@ func NewApplication(opts ApplicationOpts) (Application, error) {
 		opts.CapabilitiesRegistry = capabilities.NewRegistry(globalLogger)
 	}
 
+	// Use a recurring trigger with mock data for testing purposes
+	// TODO: proper component shutdown via srvcs()
+	_, err := capStreams.RegisterMockTrigger(globalLogger, opts.CapabilitiesRegistry)
```

Comment:
@archseer Really not wild about this: can we package up the mock as a standard capability instead? That way there won't be any code hanging around the core node to support this

Reply:
Alternatively, trigger capabilities should provide a mocked version of themselves (same output types, different implementation).